pkg/ec2metadata/ec2metadata.go

```diff
@@ -61,9 +61,10 @@ func CheckForSpotInterruptionNotice(metadataURL string) bool {
 		log.Fatalln("Could not parse time from metadata json", err.Error())
 	}
 	timeUntilInterruption := time.Now().Sub(interruptionTime)
-	if timeUntilInterruption <= (time.Duration(120) * time.Second) {
+	if timeUntilInterruption <= (time.Duration(nodeTerminationGracePeriod) * time.Second) {
 		return true
 	}
+	log.Printf("Termination notice received, but the time until interruption is %d and the node is configured to drain for %d sec. Waiting to drain...", timeUntilInterruption, nodeTerminationGracePeriod)
```

Comment:
The format string should have a "\n" at the end of it.

Reply:
updated